Automattic/newspack-plugin
# feat(stripe): add business location field

### All Submissions:






### Changes proposed in this Pull Request:



Adds a location field for Stripe settings of Reader Revenue Wizard. This is needed for Stripe payment request button (aka Apple/Google Pay). 
Also, as suggested by @thomasguillot, adds reading default values for this new field and the currency field from WooCommerce's settings. 

### How to test the changes in this Pull Request:

1. Visit the Reader Revenue wizard, set Stripe as payment platform*
2. Navigate to "Stripe Settings" tab
3. Observe a new select input in the first section – "Where is your business based?", with a default value of United States
4. Change it to something else, reload the page, observe the setting was saved
5. Switch platform to Newspack (which will use WooCommerce) and set country & currency in Reader Revenue wizard, "Address" tab
6. Remove (or temporarily rename) `newspack_stripe_data` option, switch platform back to Stripe
7. Observe the country & currency fields are same as were set in the "Address" tab for Newspack platform

\* Enabling AMP Plus by setting `NEWSPACK_AMP_PLUS_ENABLED` env. var is necessary.

### Other information:
